spec/numberformat.html

```diff
@@ -103,6 +103,22 @@
       </p>
     </emu-clause>
 
+    <emu-clause id="sec-formatnumberstring" aoid="FormatNumberString">
+      <h1>FormatNumberString ( _numberFormat_, _x_ )</h1>
+
+      <p>
+        The FormatNumberString abstract operation is called with arguments _numberFormat_ (which must be an object with fields minimumSignificantDigits, maximumSignificantDigits, minimumIntegerDigits, minimumFractionDigits and maximumFractionDigits), and _x_ (which must be a Number value), and returns _x_ as a string value with digits formatted according to the 5 formatting parameters.
+      </p>
+
+      <emu-alg>
+        1. If the _numberFormat_.[[minimumSignificantDigits]] and _numberFormat_.[[maximumSignificantDigits]] are present, then
```

Comment:
since all records with `[[initializedIntlObject]]` are formatters, the rest of the assertion is what you have in the header.